packages/apps/web-viewer-test/package.json

```diff
@@ -63,5 +59,10 @@
     "start": "react-scripts --max_old_space_size=8192 start",
     "build": "",
     "test": ""
+  },
+  "eslintConfig": {
```

Comment:
why do we need this now? Ideally all eslint config would be in the scripts package (where the pre-commit script lives)

Reply:
So theoretically we don't even need this right now, because we set `DISABLE_ESLINT_PLUGIN` in our .env file. 
But if its enabled, by default the react-app uses the default eslint rules, and it was complaining about usage of static variables and a few others, and i couldn't find the actual rule we were violating. But after setting the eslint config to point to our global repo rules, it seemed to work. 

So left this in just in case someone wanted to enable the eslint plugin in cra.

Reply:
Ok. If that is the case though, shouldn't it be added to template.json instead? The test apps won't affect the CRA template

Reply:
yea this was a concern, and something i wanted to test with the new template after this went through, but ill update accordingly now

Reply:
verified, this is not a concern by defautl if the app's eslintrc extends react-app, will update accordingly files